src/components/basicResources/CoreConcept.tsx

```diff
@@ -0,0 +1,100 @@
+import React from "react";
+import Data from "../../../public/output.json";
+import InformationCard from "../cards/InformationCard";
+import YellowCard from "../cards/YellowCard";
+import { ArcherContainer } from "react-archer";
+
+const CoreConcept = () => {
+  return (
+    <>
+      <div className="flex text-gray-500 justify-center items-center py-6 text-3xl font-semibold">
+        <h1>Core Concepts of Collaborative Design Process</h1>
+      </div>
+
+      <div className="flex gap-4 flex-col lg:flex-row md:max-w-full h-full">
+        <div className="border-4 border-dashed md:w-1/2">
+          <YellowCard
+            title={`Systems Approaches
+                to Energy Access`}
+            description={
+              "Engaging with entire systems approaches requires humanitarian practitioners to challenge and rethink how to interact with humanitarian systems that surround them. This outlines what needs to change and how."
+            }
```

Comment:
Is it possible to create may be an array outside the component which contain object of title and description? 
```const information =[{title:"title", desc: "desc"},{},.,....]```